src/main/java/org/openrewrite/javascript/JavaScriptParser.java

```diff
@@ -100,6 +100,7 @@ public Stream<SourceFile> parseInputs(Iterable<Input> inputs, @Nullable Path rel
                             } else {
                                 generator.writeString("parse-javascript-file");
                                 generator.writeString(input.getPath().toString());
+                                generator.writeString(relativeTo.toString());
```

Comment:
Whether we relativize the source file path on the Java or JS side doesn't really matter, I suppose.

Reply:
From what I learned, doing c# matters. CLI is the source of trues, so it knows what is the root dir and it better to pass it from that side